src/Magento/FunctionalTestingFramework/Console/RunTestFailedCommand.php

```diff
@@ -76,13 +76,17 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $force = $input->getOption('force');
```

Comment:
Line 54 ~ 61 are already defined in base class, and should be removed.

Reply:
Good catch, will fix.